000-vip-init.php

```diff
@@ -128,6 +128,11 @@
 // Define constants for custom VIP Go paths
 define( 'WPCOM_VIP_CLIENT_MU_PLUGIN_DIR', WP_CONTENT_DIR . '/client-mu-plugins' );
 
+// FedRAMP sites do not load Jetpack by default
+if ( method_exists( Context::class, 'is_fedramp' ) && Context::is_fedramp() && ! defined( 'VIP_JETPACK_SKIP_LOAD' ) ) {
```

Comment:
The class is probably as likely to be missing as the function, so I'd probably use `is_callable` instead of `method_exists`.

Reply:
No, class is definitely there. But lib/ will be copied over later than 000-vip-init